src/CMakeLists.txt

```diff
@@ -914,6 +918,8 @@ if(WITH_RBD)
   if(WITH_KRBD)
     add_library(krbd STATIC krbd.cc
       $<TARGET_OBJECTS:parse_secret_objs>)
+    set_target_properties(krbd PROPERTIES
```

Comment:
krbd is an convenient library, and we don't even install it. why would we need to set rpath for it?